src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java

```diff
@@ -1583,6 +1583,34 @@ void testSwitchContingency() {
         assertEquals(-596.104, postContingencyResult.getNetworkResult().getBranchResult("L2").getP2(), LoadFlowAssert.DELTA_POWER);
     }
 
+    @Test
+    void testSwitchContingency2() {
+        Network network = createNodeBreakerNetwork();
```

Comment:
Indeed there is no error in this node/breaker model because the id of the bus from the bus view and from the bus breaker view seems identical...